CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Breaking changes
 
+## 2.2.0 2023-09-11
+
+### Compatible changes
+
+- Added: State machine can now use the `:prefix` option to avoid name collision if you define multiple state machines
+  on the same model, and use state names more than once
```

Comment:
One last thing: Please use the `Unreleased` section and don't update the `version.rb` yet.

We currently have different unreleased changes and they should not try to assume a version number yet.

Version and changelog will be adjusted when we actually release.